src/_support/Step/Acceptance/Administrator/MenuItemStep.php

```diff
@@ -0,0 +1,294 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester.Step
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Step\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Step object class contains suits for MenuItem.
+ *
+ * @package  Step\Acceptance\Administrator
+ *
+ * @since    4.0.0
+ */
+class MenuItemStep extends Admin
+{
+	/**
+	 * Create A New MenuItem
+	 *
+	 * @param   string             $menuItemName       The Title Of Menu Item
+	 * @param   string             $menuItemAlias      The Alias Of Menu Item
+	 * @param   string             $title              The Title Of Category or Article or Contact or News Feed
+	 * @param   string             $menuItemType       The type of MenuItem
+	 * @param   string             $optionForMenuType  ( Archived Articles, Featured, Category Blog, etc)
+	 * @param   string             $tag                Tag
+	 *
+	 * @since   4.0.0
+	 * @return void
+	 */
+	public function menuItem($menuItemName, $menuItemAlias, $title, $menuItemType = 'Articles', $optionForMenuType = 'List All Categories', $tag = 'joomlaorg' )
```

Comment:
why not name this function createMenuItem?

Reply:
Yeah. I made that change now. Thanks